pontoon/administration/management/commands/sync_projects.py

```diff
@@ -383,6 +401,7 @@ def execute_update_db(self):
 
                     if db_translation.is_dirty():
                         self.translations_to_update.append(db_translation)
+                        self.updated_locales.add(self.locales[locale_code])
```

Comment:
Why only here? Why not also in e.g. execute_create_db() and in the "# Any existing translations that were not approved get unapproved block"?